libs/test/SalesforceReactTest/src/com/salesforce/androidsdk/reactnative/util/ReactNativeTestHost.java

```diff
@@ -77,7 +76,6 @@ protected ReactInstanceManager createReactInstanceManager() {
         ReactInstanceManagerBuilder builder = ReactInstanceManager.builder()
                 .setApplication(mApplication)
                 .setJavaScriptExecutorFactory(getJavaScriptExecutorFactory())
-                .setUIImplementationProvider(getUIImplementationProvider())
```

Comment:
This method no longer exists in RN 0.72.8 and there doesn't seem to be any documentation regarding its replacement.